tests/test_client_options.py

```diff
@@ -3,41 +3,41 @@
 from supabase.lib.client_options import ClientOptions
 
 
-def test__client_options__replace__returns_updated_options():
-    storage = SyncMemoryStorage()
-    storage.set_item("key", "value")
-    options = ClientOptions(
-        schema="schema",
-        headers={"key": "value"},
-        auto_refresh_token=False,
-        persist_session=False,
-        storage=storage,
-        realtime={"key": "value"},
-    )
-
-    actual = options.replace(schema="new schema")
-    expected = ClientOptions(
-        schema="new schema",
-        headers={"key": "value"},
-        auto_refresh_token=False,
-        persist_session=False,
-        storage=storage,
-        realtime={"key": "value"},
-    )
-
-    assert actual == expected
-
-
-def test__client_options__replace__updates_only_new_options():
-    # Arrange
-    storage = SyncMemoryStorage()
-    storage.set_item("key", "value")
-    options = ClientOptions(storage=storage)
-    new_options = options.replace()
-
-    # Act
-    new_options.storage.set_item("key", "new_value")
-
-    # Assert
-    assert options.storage.get_item("key") == "new_value"
-    assert new_options.storage.get_item("key") == "new_value"
+class TestClientOptions:
```

Comment:
Hmm I don't get the rationale here - can you elaborate how the class makes the tests more readable?

Reply:
Both the methods were for testing client options so instead of prefixing the word `test_client_options` again and again it was better to keep both inside a class which broadly describes their purpose.

Reply:
Fair enough I guess